src/content/docs/cheerpj3/12-reference/00-cheerpjInit.md

```diff
@@ -154,6 +154,24 @@ The first time an operation is requested, the user will be asked for permission
 
 > This mode is experimental and might show a few inconsistencies in some browsers. [`See more`](https://caniuse.com/mdn-api_permissions_permission_clipboard-read).
 
+### `beepCallback`
+
+```ts
+beepCallback?: () => void;
+```
+
+This callback runs when `java.awt.Toolkit.getDefaultToolkit().beep()` is called in java.
+
+Example of usage:
+
+```js
+cheerpjInit({
+	beepCallback: function () {
```

Comment:
Let's be consistent with the rest of the file; use beepCallback() {...} shorthand syntax

Reply:
NVM we're not consistent anyway